validphys2/src/validphys/scripts/vp_comparefits.py

```diff
@@ -223,8 +223,13 @@ def complete_mapping(self):
                     'unpolarized_bc': {'from_': 'positivity_bound'},
                 }
             )
+        are_the_same = self.check_identical_theory_cuts_covmat()
+        if are_the_same:
+            log.info("Adding mismatched datasets page: identical theory cuts/covmat detected")
+            autosettings["template"] = "report_mismatched.md"
```

Comment:
Instead of changing the whole template (and then needing a whole new report) you can change directly the mismatched_report part.

e.g., if `are_the_same` is `True` then the `template` part under `mismatched_report` stays, otherwise you drop it and change it to `template_text: "Difference settings, mismatched page cannot be added".